lib/page-object.rb

```diff
@@ -9,6 +9,11 @@
 require 'page-object/indexed_properties'
 require 'page-object/widgets'
 
+require 'page-object/platforms/watir_webdriver/element'
```

Comment:
I figured putting all these requires upfront wouldn't hurt anything and it keeps them out of the initialize method for elements. I ran a set of benchmarks on collecting sets of elements with page-object, watir, and selenium, and page object took 10 times as long, until I moved these statements.